example/lib/ui/push_notifications/push_notifications_sliver.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+
+import '../../push_notification_service.dart';
+import 'push_notifications_activation_sliver.dart';
+import 'push_notifications_publishing_sliver.dart';
+import 'push_notifications_subscriptions_sliver.dart';
+
+class PushNotificationsSliver extends StatelessWidget {
+  late final PushNotificationService _pushNotificationService;
+
+  PushNotificationsSliver(this._pushNotificationService, {Key? key})
+      : super(key: key);
+
+  Widget buildCreateAblyClientText() => StreamBuilder(
+        stream: _pushNotificationService.hasPushChannelStream,
+        builder: (context, snapshot) {
+          if (!snapshot.hasData || snapshot.hasData && snapshot.data == false) {
+            return Padding(
+              padding: const EdgeInsets.only(bottom: 16),
+              child: RichText(
+                  text: const TextSpan(children: [
+                TextSpan(
+                    text: 'Warning: ',
+                    style: TextStyle(
+                        color: Colors.red, fontWeight: FontWeight.bold)),
+                TextSpan(
+                    text: 'Create an Ably realtime or rest client above',
+                    style: TextStyle(color: Colors.black))
+              ])),
+            );
+          }
+
+          return const SizedBox.shrink();
+        },
+      );
+
+  @override
+  Widget build(BuildContext context) => Container(
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.stretch,
+          children: [
+            const Padding(
+              padding: EdgeInsets.only(bottom: 16),
+              child: Text(
+                'Push Notifications',
+                style: TextStyle(fontSize: 24, fontWeight: FontWeight.bold),
+              ),
+            ),
+            buildCreateAblyClientText(),
+            buildSummaryText(),
+            PushNotificationsActivationSliver(_pushNotificationService),
+            PushNotificationsSubscriptionsSliver(_pushNotificationService),
+            PushNotificationsPublishingSliver(_pushNotificationService),
+            // TODO Implement Push Admin API and then use the following:
```

Comment:
Is there an issue to track this?

Reply:
Yes, it's here https://github.com/ably/ably-flutter/issues/109

I left some related files because that is where they would go if they would be implemented.

Reply:
I thought about it, and there is no point keeping a few fields for a feature that is not yet implemented (since it is a "server side" feature as described in the docs). It will also go out of sync with decisions.

8cd1585b6f3d9bcacff42eb536c05d30fe42f7be removed the fields and comments.

Please see and use https://github.com/ably/ably-flutter/issues/109 to track work on this.